.github/workflows/dco.yaml

```diff
@@ -0,0 +1,18 @@
+name: Developer Certificate of Origin Check
+
```

Comment:
Hey @salyh the DCO check is done by an app https://github.com/dcoapp that is in boarded across https://github.com/opensearch-project org that takes care of DCO check for all repositories part of the org, so we should be good removing this CI check.
Adding @bbarani 
Thanks

Reply:
I saw the check in many repos (https://github.com/search?q=org%3Aopensearch-project%20tim-actions%2Fdco&type=code) and I was not aware of the global installed app.

I will remove it. If still needed we can add it later again.